src/rpc/blockchain.cpp

```diff
@@ -1633,13 +1633,19 @@ static RPCHelpMan getchaintxstats()
                     RPCResult::Type::OBJ, "", "",
                     {
                         {RPCResult::Type::NUM_TIME, "time", "The timestamp for the final block in the window, expressed in " + UNIX_EPOCH_TIME},
-                        {RPCResult::Type::NUM, "txcount", "The total number of transactions in the chain up to that point"},
+                        {RPCResult::Type::NUM, "txcount", /*optional=*/true,
```

Comment:
Not returning a value here might break downstream projects if they are unlucky since this wasn't optional before. Not sure if deprecation is necessary here but a more safe approach could be to just return 0 if it is unknown. I don't have a strong opinion on it but usually, we are pretty cautious with a change like this.

Reply:
Yeah, changing the return value wouldn't be much of a concern, but a missing key has higher potential to crash something.

> When evaluating whether this is a breaking change, I don't think it can affect any downstream projects, because Assume-Utxo is an incomplete, experimental, regtest-only feature. 

(*testnet/regtest-only) I'm pretty much with you here but [others might not be](https://github.com/bitcoin/bitcoin/pull/29612#pullrequestreview-1974818797) though the utxo set dump might also be used in a different context. Feel free to leave the approach as is, I am fine with it.

Reply:
We're breaking the format anyway with #29612 so might as well introduce other breaking changes now.

Reply:
Both are listed as critical for mainnet release in #29616. If one needs a breaking change, it's fine for the other to also make one.